packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -4829,4 +4838,1090 @@ describe("IModelTransformerHub", () => {
+      sinon
+        .stub(transformer, "shouldExportElement")
+        .callsFake((_sourceElement) => true);
+      sinon
+        .stub(transformer, "addCustomChanges")
+        .callsFake(async (sourceDbChanges) => {
+          expect(
+            sourceDbChanges.hasChanges,
+            "there should be only custom changes"
+          ).to.be.false;
+          await sourceDbChanges.addCustomModelChange(
+            "Inserted",
+            childDrawing2.id!
+          );
+        });
+      await transformer.process();
+      await pushChanges(
+        targetDb,
+        "Transformation 2: add second previously excluded child element"
+      );
+      // Assert
+      assertModelExistsByName(targetDb, [
+        "DL",
+        "ParentDrawing1",
+        "ParentDrawing2",
+        "ChildDrawing1",
+        "ChildDrawing2",
+      ]);
+      assertElementsExistByCode(targetDb, [
+        parentDrawing1,
+        childDrawing1,
+        childDrawing2,
+      ]);
+      assertElementHasExpectedAspectCount(
+        targetDb,
+        childDrawing2.federationGuid!,
+        1
+      );
+      expect(
+        IModelTestUtils.count(targetDb, ElementGroupsMembers.classFullName)
+      ).to.be.equal(2);
+
+      // === Transformation 4: `process changes` transformation to delete first child element's sub model  ===
+      transformer = new CustomChangesTransformer(sourceDb, targetDb, true);
+      sinon
+        .stub(transformer, "shouldExportElement")
+        .callsFake((_sourceElement) => true);
+      sinon
+        .stub(transformer, "addCustomChanges")
+        .callsFake(async (sourceDbChanges) => {
+          expect(
+            sourceDbChanges.hasChanges,
+            "there should be only custom changes"
+          ).to.be.false;
+          await sourceDbChanges.addCustomModelChange(
+            "Deleted",
+            childDrawing1.id!
+          );
+        });
+      await transformer.process();
+      await pushChanges(
+        targetDb,
+        "Transformation 3: delete first child element's submodel"
+      );
+      assertModelExistsByName(targetDb, [
+        "DL",
+        "ParentDrawing1",
+        "ParentDrawing2",
+        "ChildDrawing2",
+      ]);
+      assertElementsExistByCode(targetDb, [parentDrawing1, childDrawing2]);
+      assertElementsDoNotExistByCode(targetDb, [childDrawing1]);
+      expect(
+        IModelTestUtils.count(targetDb, ElementGroupsMembers.classFullName)
+      ).to.be.equal(1);
+    });
+
+    it("should update exported data correctly when custom changes are registered for elements", async function () {
+      // Prepare source
+      const sourceSubjectId = Subject.insert(
+        sourceDb,
+        IModel.rootSubjectId,
+        "S1"
+      );
+      const categoryId1 = SpatialCategory.insert(
+        sourceDb,
+        IModel.dictionaryId,
+        "C1",
+        {}
+      );
+      const physicalModel1Id = PhysicalModel.insert(
```

Comment:
@DanRod1999  Got question from reviewer, why these are formatted so strange, and that those could look better if they were one-liners. But it is re-formatted to this on commit, I think "linter" or "prettifier" does this ? Does it look as you want it to look? ( spread across multiple lines instead of being a one-liner)?